core/backend/src/BriefcaseManager.ts

```diff
@@ -416,6 +416,10 @@ export class BriefcaseManager {
     for (const changeSet of changeSets)
       await this.applySingleChangeset(db, changeSet, processOption);
 
+    // clear statement cache if there was at least on schema changeset applied

```

Comment:
this is not necessary. We call `clearCaches` in `applySingleChangeset`.

Reply:
fixed. _i think when i did this change a while back it was not cleared_